.travis.yml

```diff
@@ -131,7 +131,7 @@ jobs:
       go: $TRAVIS_GO_VERSION
       stage: test
     - os: osx
-      env: TARGETS="TEST_ENVIRONMENT=0 -C x-pack/functionbeat testsuite"
+      env: TARGETS="CGO_ENABLED=0 TEST_ENVIRONMENT=0 -C x-pack/functionbeat testsuite"
```

Comment:
I suggest to set `params.CGO` to `false` in the `magefile` of functionbeat. We don't need cgo at all in case of Functionbeat.

Reply:
Both suggestions are valid to solve this building issue but if functionbeat doesn't need cgo then I think it is a best approach to set  `params.CGO` to `false` because it is bounded to functionbeat and less invasive than modify travis configuration.  WDYT @ycombinator

Reply:
Yep, I agree that disabling cgo in the functionbeat magefile is better because it's not just for Travis and, ultimately, Functionbeat doesn't need cgo.